addons/mail/models/mail_presence.py

```diff
@@ -22,8 +22,8 @@ class MailPresence(models.Model):
     _description = "User/Guest Presence"
     _log_access = False
 
-    user_id = fields.Many2one("res.users", "Users", ondelete="cascade")
-    guest_id = fields.Many2one("mail.guest", "Guest", ondelete="cascade")
+    user_id = fields.Many2one("res.users", "Users", index='btree_not_null', ondelete="cascade")
```

Comment:
Isn't it already indexed through `_user_unique`?

Reply:
Yup, good catch, indeed they are essentially unique btree_not_null indexes already.